api/framework/admin-announcements/bundle.md

```diff
@@ -0,0 +1,54 @@
+# admin-announcements
```

Comment:
I think the admin bundles should all be under the admin "namespace" instead of framework. At least these new ones.